python/py-pygithub/Portfile

```diff
@@ -1,31 +1,38 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github 1.0
```

Comment:
Why are you switching away from GitHub?

Reply:
The github release has a lot of files that aren't required for the build, making the download about 20x larger (`~158046` vs `~3123996`). It's still tiny by today's standard though ;)

More than happy to revert back if preferred. 👍 I would then also add `github.tarball_from archive`.

Reply:
The smaller download size is actually a good point! :+1: